packages/client/src/main.tsx

```diff
@@ -7,32 +7,38 @@ import { store } from '@store'
 
 const initServiceWorkers = () => {
     if ('serviceWorker' in navigator) {
-        window.addEventListener('load', () => {
-            navigator.serviceWorker
-                .register('/sw.js', { scope: '/' })
-                .then(registration => {
-                    const data = {
-                        type: 'CACHE_URLS',
-                        payload: [
-                            location.href,
-                            ...performance
-                                .getEntriesByType('resource')
-                                .map(r => r.name),
-                        ],
-                    }
-                    registration?.installing?.postMessage(data)
-                    console.log('Service worker registered: ', registration)
-                })
-                .catch(error => {
-                    console.error('Service worker registration failed: ', error)
-                })
-        })
+        if (typeof window !== 'undefined') {
```

Comment:
Лучше обернуть этим условием вызов функции initServiceWorkers

```
if (typeof window !== 'undefined') {
   initServiceWorkers()
}
```

Либо убрать лишнюю вложенность сделав обратное условие 
```suggestion
        if (typeof window === 'undefined') {
            return
        }
```